modules/client-configuration/pages/snippets/trust_gpg.adoc

```diff
@@ -32,5 +75,6 @@ You do not need to delete any previously stored keys.
 ====
 Trusting a GPG key is important for security on clients. 
 It is the task of the admin to decide which keys are needed and can be trusted.
-Trusting the key is done manually,  either by writing a Salt state or adding the keys to the bootstrap script.
+As a Software Channel cannot be used when the GPG key is not trusted, the decision of assigning a channel
+to a client is bound to the decission of trusting the key.
```

Comment:
```suggestion
Because a software channel cannot be used when the GPG key is not trusted, the decision of assigning a channel to a client depends on the decision of trusting the key.
```